src/OpenApi/Generator/EndpointGenerator.php

```diff
@@ -362,14 +363,21 @@ private function getOptionsResolverMethod(Operation $operation, string $paramete
 
     private function getGetBody(Operation $operation, Context $context): Stmt\ClassMethod
     {
+        $stmts = [];
+        
+        if ($operation->getOperation()->getRequestBody() === null ||
```

Comment:
Why `null` here ?

Reply:
The following method where the request body is passed into allows null.